### ISSUE TITLE
Deploy with resources fails on Linux

### ISSUE THREAD

**Issue by greenfork** [factor/factor#3077]
Reproduction:

```
cd factor-path/work
git clone https://github.com/JosephOziel/bitguessr.git
factor  # run listener
```

In listener:
```
"bitguessr" deploy
```

Outputs:

```
1: Note:
Added "tools.deploy" vocabulary to search path
Loading resource:work/bitguessr/bitguessr.factor
Loading resource:extra/raylib/raylib.factor
Loading resource:extra/raylib/util/util.factor
Loading resource:extra/raylib/raylib-docs.factor
{
    "bitguessr/bitguessr"
    "-pic=0"
    "-i=/home/grfork/.cache/factor/staging.math-threads-compiler-i..."
    "-vocab-manifest-out=/tmp/factor-temp/vocab-manifest-bitguessr"
    "-deploy-vocab=bitguessr"
    "-deploy-config=/tmp/factor-temp/deploy-config-bitguessr"
    "-output-image=/tmp/factor-temp/bitguessr.image"
    "-resource-path=/home/grfork/reps/factor/"
    "-run=tools.deploy.shaker"
}
Loading resource:basis/tools/deploy/shaker/shaker.factor
Loading resource:basis/tools/deploy/config/config.factor
Deploying bitguessr...
Loading /tmp/factor-temp/deploy-config-bitguessr
Loading resource:basis/debugger/debugger.factor
Loading resource:basis/fixups/fixups.factor
Loading resource:basis/io/styles/styles.factor
Loading resource:basis/colors/colors.factor
Loading resource:basis/ascii/ascii.factor
Loading resource:basis/delegate/protocols/protocols.factor
Loading resource:basis/strings/tables/tables.factor
Loading resource:basis/prettyprint/prettyprint.factor
Loading resource:basis/prettyprint/config/config.factor
Loading resource:basis/prettyprint/sections/sections.factor
Loading resource:basis/vocabs/prettyprint/prettyprint.factor
Loading resource:basis/prettyprint/backend/backend.factor
Loading resource:basis/lists/lists.factor
Loading resource:basis/prettyprint/stylesheet/stylesheet.factor
Loading resource:basis/hashtables/wrapped/prettyprint/prettyprint.factor
Loading resource:basis/hashtables/identity/prettyprint/prettyprint.factor
Loading resource:basis/locals/definitions/definitions.factor
Loading resource:basis/see/see.factor
Loading resource:basis/locals/prettyprint/prettyprint.factor
Loading resource:basis/dlists/prettyprint/prettyprint.factor
Loading resource:basis/specialized-arrays/prettyprint/prettyprint.factor
Loading resource:basis/classes/struct/prettyprint/prettyprint.factor
Loading resource:basis/alien/prettyprint/prettyprint.factor
Loading resource:basis/mirrors/mirrors.factor
Loading resource:basis/hashtables/identity/mirrors/mirrors.factor
Loading resource:basis/specialized-arrays/mirrors/mirrors.factor
Loading resource:basis/linked-assocs/prettyprint/prettyprint.factor
Loading resource:basis/bit-arrays/prettyprint/prettyprint.factor
Loading resource:basis/compiler/prettyprint/prettyprint.factor
Loading resource:basis/typed/prettyprint/prettyprint.factor
Loading resource:basis/colors/prettyprint/prettyprint.factor
Loading resource:basis/debugger/threads/threads.factor
Loading resource:basis/debugger/unix/unix.factor
Loading resource:basis/source-files/errors/debugger/debugger.factor
Loading resource:basis/command-line/debugger/debugger.factor
Loading resource:basis/stack-checker/errors/prettyprint/prettyprint.factor
Loading resource:basis/unix/debugger/debugger.factor
Loading resource:basis/tools/errors/errors.factor
Loading resource:basis/inspector/inspector.factor
Loading resource:work/bitguessr/bitguessr.factor
Loading resource:basis/formatting/formatting.factor
Loading resource:basis/calendar/english/english.factor
Loading resource:basis/multiline/multiline.factor
Loading resource:basis/peg/ebnf/ebnf.factor
Loading resource:basis/peg/peg.factor
Loading resource:basis/unicode/unicode.factor
Loading resource:basis/interval-maps/interval-maps.factor
Loading resource:basis/unicode/breaks/breaks.factor
Loading resource:basis/simple-flat-file/simple-flat-file.factor
Loading resource:basis/unicode/categories/categories.factor
Loading resource:basis/unicode/data/data.factor
Loading resource:basis/interval-sets/interval-sets.factor
Loading resource:basis/unicode/normalize/normalize.factor
Loading resource:basis/sorting/insertion/insertion.factor
Loading resource:basis/unicode/case/case.factor
Loading resource:basis/unicode/collation/collation.factor
Loading resource:basis/unicode/script/script.factor
Loading resource:basis/peg/debugger/debugger.factor
Loading resource:basis/peg/parsers/parsers.factor
Loading resource:basis/peg/search/search.factor
Loading resource:basis/random/random.factor
Loading resource:basis/random/unix/unix.factor
Loading resource:basis/random/mersenne-twister/mersenne-twister.factor
Loading resource:extra/raylib/raylib.factor
Loading resource:extra/raylib/util/util.factor
Writing vocabulary manifest to /tmp/factor-temp/vocab-manifest-bitguessr
Preparing deployed libraries
Stripping manual memory management debug code
Stripping destructor debug code
Stripping stack effect checking from call( and execute(
Stripping startup hooks
Stripping C I/O
Finding megamorphic caches
Compressing objects
Compressing quotations
Stripping word properties
Stripping symbolic word definitions
Clearing memoized word caches
Clearing megamorphic caches
Saving final image
```

and fails with this error:

```
Unix system call 'mkdir' failed:
Not a directory (20)
It was called with the following arguments:
"/home/grfork/reps/factor/bitguessr/bitguessr/_resources"
511
```

**Comment by greenfork** [comment]
Probably related https://github.com/factor/factor/issues/678

**Comment by mrjbq7** [comment]
Fixed in #3080.
